version/1/2/0/code_of_conduct.md

```diff
@@ -0,0 +1,23 @@
+# Contributor Code of Conduct
+
+As contributors and maintainers of this project, we pledge to respect all people who contribute through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, nationality, or political affiliation.
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexual language or imagery
+* Derogatory comments
+* Personal attacks
+* Trolling or insulting comments
+* Public or private harassment
+* Publishing other's private information, such as physical or electronic addresses, without permission
+* Other unethical or unprofessional conduct.
```

Comment:
Also, 'ethics' usually refers to 'professional ethics', and well, we aren't a profession.